Documentation/Contribution/Index.rst

```diff
@@ -0,0 +1,36 @@
+Contribution
+============
+
+In order to ease contributions there is a preconfigured ddev environment.
+
+System Requirements
+-------------------
+
+1) DDEV and Docker installed
+
+For an installation guide for your platform see
+
+* https://ddev.com/get-started/ and
+* https://ddev.readthedocs.io/en/stable/
+
+2) `git lfs` installed
+
+`git lfs` is used to handle larger files, like DB Dump and fileadmin, for the initial install.
+
+See here how to install and work with it:
+
+https://github.com/git-lfs/git-lfs
+
+Setup the Development Environment
+---------------------------------
+
+1) Clone the repo, as you do with any other project
```

Comment:
Which repo do you mean? I'm a bit lost here ;-)

Reply:
This is a left over. I planned to provide a local dev environment within this repo. Unfortunately it is not possible to activate git-lfs on forked repos. So I removed it locally, but forgot to remove the documentation. 

It was quite similar to your seperate repo at https://github.com/albig/ddev-ods-osm

Sorry for that.